packages/cactus-common/package.json

```diff
@@ -69,11 +69,15 @@
     "loglevel-plugin-prefix": "0.8.4",
     "secp256k1": "4.0.2",
     "sha3": "2.1.3",
-    "typescript-optional": "2.0.1"
+    "typescript-optional": "2.0.1",
+    "uuid": "^8.3.1"
```

Comment:
@gabrielsoaresm94 Because the uuid package is only used by the tests: Please move it to the `devDependencies` of the package.json from the `dependencies` to reflect that.

Reply:
@patrickmn Done, I switched to `devDependencies`.